test/test_prototype_transforms_functional.py

```diff
@@ -421,6 +421,14 @@ def center_crop_bounding_box():
         )
 
 
+def center_crop_segmentation_mask():
+    for mask, output_size in itertools.product(
+        make_segmentation_masks(image_sizes=((16, 16), (7, 33), (31, 9)), extra_dims=((), (4,), (2, 3))),
```

Comment:
Nit: it is a good idea to be explicit about the `image_sizes` here due to their relation to the crop size, but the same is not true for the `extra_dims`:

```suggestion
        make_segmentation_masks(image_sizes=((16, 16), (7, 33), (31, 9))),
```

On the other hand it also doesn't hurt to be explicit about them. Feel free to leave as is.